src/cmd/api/controllers/video_upload_handler.go

```diff
@@ -112,10 +161,28 @@ func (v VideoUploadHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 
 	if err = v.AmqpClient.Publish(events.VideoUploaded, videoData); err != nil {
 		log.Error("Unable to publish on Amqp client ", err)
+
+		videoEncodeFailed(videoCreated, v.MariadbClient)
```

Comment:
No here we are not sure whether the encoding did fail or not. The log itslef is sufficient, we will need to treat the status of failed encoding elsewhere (i.e. not in this controller)